<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frm_single_card_import</class>
 <widget class="QDialog" name="frm_single_card_import">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btn_cancel_frm_single_import">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>550</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_date_single">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>150</y>
     <width>111</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_reset_path_single">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>330</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_mode_single">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>120</y>
     <width>101</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Mode</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_mode_single">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>150</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lb_band_single">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>120</y>
     <width>101</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Band</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_select_path_single">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>330</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_singlecard_import">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>330</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_date_single">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>120</y>
     <width>127</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Date</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_callsign_single">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>150</y>
     <width>111</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lb_path_singlecard">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>300</y>
     <width>301</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Path for Single Card Import</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_path_singlecard_import">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>330</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="lb_single_card_upload_section">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>601</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Single Card Import</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lb_callsign_single">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>127</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Callsign</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_band_single">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>150</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
